Add JPD

louiss0 opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue? #271690

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.10 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0)?


---



I want to add my package called JPD! its a CLI for managing JavaScript Packages! I'm sorry for now completing this list, but I'm on Linux. Is there a way for me to test it here, or do I have to use windows? This is my first time contributing this repo. I read the paper, and I was told that I'd be prompted to sign! I'm willing to do what I can to submit this package properly! I  



 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/271693)

louiss0 committed:
feat: place jpd manifest in the repo

louiss0 committed:
Merge branch 'microsoft:master' into master

louiss0 committed:
Merge branch 'microsoft:master' into master

Dragon1573 commented:
Hi @louiss0, 👋🏼 glad to see you resolving your own issue.

> [!TIP]
> 
> It's recommended linking the pull request to that issue using ["close keywords"](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue). You can edit your description like follows.
> 
> ```markdown
> Checklist for Pull Requests
> 
> - [x] Is there a linked Issue? Resolve #271690
> ```

louiss0 commented:
> Hello @louiss0,
> 
> Please verify the manifest file is compliant with the package manager [1.10.0 manifest specification](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0).
> 
> Make sure the ID is of the form publisher.appname and that the folder structure is manifests\partition\publisher\appname\version.
> 
> > Note: The path and "PackageIdentifier" are case sensitive.
> 
> Be sure to use a tool like [VSCode](code.visualstudio.com) to make sure the manifest YAML syntax is correct. You can also validate your manifest using the `winget validate` command.
> 
> For details on the specific error, see the details link below in the build pipeline.
> 
> Template: msftbot/validationError/manifest/content

I changed the output I got it from the wrong place I'm sorry

Dragon1573 reviewed (changes requested):
✅ After removing `ReleaseNotes`, manifests works as expected.

<img width="1718" height="1454" alt="Image" src="https://github.com/user-attachments/assets/5f775da5-c1fe-44a3-b7ce-860bf17dda52" />
<img width="1718" height="1014" alt="Image" src="https://github.com/user-attachments/assets/99babede-a6c0-4f0c-a8c6-3ff2aeeea6b9" />

louiss0 committed:
fix: replace release notes with release url

The release notes are too much for this specific release it's best to use a URL this time!

louiss0 committed:
Merge branch 'microsoft:master' into master

louiss0 committed:
fix: use correct issue URL

The publisher URL is incorrect it needs to refer to the project's repo!
